runners/core-java/src/main/java/org/apache/beam/runners/core/metrics/ExecutionStateTracker.java

```diff
@@ -321,6 +354,13 @@ public void takeSample(long millisSinceLastSample) {
     }
   }
 
+  protected void reportBundleLull(String customLogMessage) {
+    if (currentState != null) {
+      currentState.reportBundleLull(trackedThread, customLogMessage, millisSinceBundleStart);
```

Comment:
example is here. This wouldn't log if there wasn't a current state. But we're logging about the bundle, not anything about the particular state.